src/fixture-02.ts

```diff
@@ -0,0 +1,59 @@
+import { test as base } from "@playwright/test";
+
+const xpathUserName = '//input[@id="user_login"]';
+const xpathPassWord = '//input[@id="user_pass"]';
+const xpathBtnSubmit = '//input[@id="wp-submit"]';
+const xpathMenuTags = "//a[text()='Tags']";
+const xpathBtnAddNewTag = "//input[@id='submit']";
+const xpathInputName = "//input[@id='tag-name']";
+const xpathInputSlug = "//input[@id='tag-slug']";
+
+const myTag = base.extend<{
+  createTag: (tagName: string, slugName?: string) => Promise<void>;
+}>({
+
+  page: async ({ page }, use) => {
```

Comment:
```suggestion
  myTag: async ({ page }, use) => {
```
khai báo fixture ở đây mới đúng